hooks/node_handlers/base_import_handler.py

```diff
@@ -37,11 +40,54 @@ class ImportNodeHandler(HookBaseClass):
     SGTK_BROWSE = "sgtk_browse"
     SGTK_PUBLISH_DATA = "sgtk_publish_data"
     SGTK_LAST_USED = "sgtk_last_used"
+    SGTK_PATH_PUBLISH = "sgtk_path_selection1"
+    SGTK_PATH_WORK = "sgtk_path_selection2"
+    SGTK_PATH_FILE = "sgtk_path_selection3"
+    SGTK_PATH_SELECTION = "sgtk_path_selection11"
+    SGTK_WORK_FILE_DATA = "sgtk_work_file_data"
+    SGTK_NODE_NAME = "sgtk_node_name"
+    SGTK_NODE_PARM = "sgtk_node_parm"
+    SGTK_WORK_VERSION = "sgtk_work_version"
+    SGTK_WORK_REFRESH_VERSIONS = "sgtk_work_refresh_versions"
+    SGTK_WORK_RESOLVED_VERSION = "sgtk_work_resolved_version"
+    SGTK_FILE_DATA = "sgtk_file_data"
+    SGTK_FILE_PATH = "sgtk_file_path"
+    SGTK_FILE_VERSION = "sgtk_file_version"
+    SGTK_FILE_REFRESH_VERSIONS = "sgtk_file_refresh_versions"
+    SGTK_FILE_RESOLVED_VERSION = "sgtk_file_resolved_version"
+    SGTK_CURRENT_TAB = "sgtk_current_tab"
+
+    DEFAULT_WORK_FILE_DATA = json.dumps(
+        {
+            "node": None,
+            "all_parms": [],
+            "parm": None,
+            "all_versions": [],
+            "current_version": LATEST_POLICY,
+        }
+    )
+    DEFAULT_FILE_DATA = json.dumps(
+        {
+            "path": None,
+            "all_versions": [],
+            "current_version": LATEST_POLICY,
+        }
+    )
+    FILECHOOSER_PATTERN = "*"
+    HOU_FILE_TYPE = hou.fileType.Any
 
     VALID_FILE_TYPES = "valid_file_types"
 
     ACCEPTS_MULTI_SELECTION = False
 
+    PUBLISH, WORK, FILE = range(3)
+
+    WORK_VERSION_TEMPLATE = r"v{}"
+    WORK_VERSION_REGEX = WORK_VERSION_TEMPLATE.format(r"(\d+)")
+    SEQUENCE_REGEX = r"-9999"  # This is a slight hack
```

Comment:
IGNORABLE:

- Explicit expansion
- No real need for "r" strings unless there are "\" in string

```suggestion
    PUBLISH, WORK, FILE = 0, 1, 2

    WORK_VERSION_TEMPLATE = "v{}"
    WORK_VERSION_REGEX = WORK_VERSION_TEMPLATE.format(r"(\d+)")
    SEQUENCE_REGEX = "-9999"  # This is a slight hack
```